smart_card_connector_app/src/_locales/en/messages.json

```diff
@@ -35,6 +35,10 @@
     "message": "ADD READER",
     "description": "Title of the button that can be used to open a dialog for adding a new Smart Card reader"
```

Comment:
nit: I would make it explicit here that this only appears when there are no readers yet. Generally, these descriptions exist to give context to the translators, so giving more details is helpful, and this info might make a difference for the translation in some languages.